packages/link-modules/src/links.ts

```diff
@@ -14,6 +14,12 @@ export const LINKS = {
     Modules.PRICING,
     "price_set_id"
   ),
+  CartRegion: composeLinkName(
```

Comment:
q: should this be deleted now that it's read-only?

Reply:
I don't completely understand what this file does, but just saw the reference to a table name in the `composeLinkName` function. Since there's no join-table anymore, I thought it might not be needed anymore?

Reply:
this file is only a definition of all known links among Medusa modules.
We could bring these link names to the link definition itself, but there is the case where someone needs to replace a link module by a custom module and can use this to reference the original module name.

Reply:
But @srindom is correct in that we don't need it for read-only links right?

Reply:
yes he is. I was just explaing what the file is haha. sorry.

Reply:
read only links, do not generate a module. The definition is only used to compsoe the graph.

Reply:
> yes he is. I was just explaing what the file is haha. sorry.

All good. I needed a refresher too, so thx! :D